services/horizon/internal/app.go

```diff
@@ -25,9 +24,10 @@ import (
 	"github.com/kinecosystem/go/support/app"
 	"github.com/kinecosystem/go/support/db"
 	"github.com/kinecosystem/go/support/log"
+	"github.com/rcrowley/go-metrics"
```

Comment:
- this renames `metrics` to `go-metrics` without renaming where this package is called in this file. how did this pass compilation?
- if possible i think it's better to avoid making unnecessary changes to important files like `app.go`. we'll have to solve merge conflicts when fetching updates from stellar

Reply:
IDE is doing it automatically, I guess that if the right use of it (gofmt)